db/migrate/20170410135556_add_csv_to_imports.rb

```diff
@@ -0,0 +1,6 @@
+class AddCsvToImports < ActiveRecord::Migration[5.0]
+  def change
+    add_column :imports, :csv, :string
+    remove_column :imports, :uri
```

Comment:
Wow, trickery! But impressive that they can pack all that info into a string column. 😝 

Yeah, I'm good with `file` or something like that. I think the call site would be nice like that:

```
import.file.url
```